pymatgen/io/lobster/outputs.py

```diff
@@ -408,6 +409,113 @@ def icohpcollection(self):
         return self._icohpcollection
 
 
+class Ncicobilist:
```

Comment:
Why not pascal case `NciCobiList` here? Easier to read imo.

Reply:
Here I wanted to stick to only capitalizing the first letter like in  `class Icohplist` 
For me `NciCobilist` is not very readable as the method referring to is NcICOBI, so maybe we can go with `NcIcobilist` or `ncIcobilist` ?

Reply:
Hi @janosh, please let me know what you think about this. If it's not ok, I'm gonna adopt your suggestion.

Reply:
I still prefer consistency with the rest of `pymatgen` which uses PascalCase for all class names. E.g. we have `PotcarSingle` and not `POTCARSingle` for that reason. But maybe another maintainer wants to chime in here.

Reply:
(It does not have to be consistent with the older Lobster class names.)

Reply:
@janosh, ok it's done :)))